frontend2/src/pages/prompt/PromptEntryPage.tsx

```
@@ -11,49 +11,49 @@ export function PromptEntryPage() {
   const navigate = useNavigate();
 
   if (!isValidWorkspaceId) {
-    return <div className="p-8 text-gray-500">유효하지 않은 워크스페이스입니다.</div>;
+    return <div className="p-8 text-[var(--text-secondary)]">유효하지 않은 워크스페이스입니다.</div>;
   }
 
   const workspaceId = parsedWorkspaceId;
   const basePath = orgId ? `/orgs/${orgId}/workspaces/${workspaceId}` : `/workspaces/${workspaceId}`;
 
   const { data: prompts, isLoading } = useQuery({
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)